pet.js

```diff
@@ -1,14 +1,36 @@
-class Pet {
-    constructor() {
-        // set up basic stats
-        this.health = 100;
+var petName = document.getElementById("nameInput").value;
+
+class petClass {
```

Comment:
Class names should start with a capital - we can leave this as Pet as this shouldn't conflict with anything in the main.js